docs/guides/api-gateway/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+pagination_next: guides/device-gateway/index
+pagination_prev: errors/index
+sidebar_position: 2
+title: API Gateway
+---
+
+# API Gateway
+
+This section provides guides for using ngrok as an API Gateway.
+
+| Name                                                                | Description                                                                                                                       |
+| ------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| [Deliver and secure ingress for APIs in Kubernetes](kubernetes.mdx) | Deploy ngrok's Kubernetes Operator onto your cluster for unified ingress and advanced traffic shaping with K8s-native interfaces. |
```

Comment:
Can you remove the period after "interfaces" to be consistent with the other guides landing pages?